src/main/java/com/esri/samples/portal/integrated_windows_authentication/README.md

```diff
@@ -6,43 +6,37 @@ Use Windows credentials to access services hosted on a portal secured with Integ
 
 ## Use case
```

Comment:
This should be HTML format, unless dev branch is at the stage where we're ok to start putting in samples as .md?